CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -0,0 +1,99 @@
+# Release notes for 1.10.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.9.0
+
+## Changes by Kind
+
+### Feature
+
+- Add Modify Volume Support for VolumeAttributesClass, this requires:
+  1. The feature gate VolumeAttributesClass and API to be enabled in Kubernetes cluster
```

Comment:
Also note this requires a 1.29 cluster

Reply:
It is listed in ### Uncategorized

Do we want to categorize it as feature from now on?

Reply:
Updated.